plugins/coreboot/README.md

```diff
@@ -0,0 +1,62 @@
+coreboot
+========
+
+Introduction
+------------
+
+This plugin uses the `flashrom` plugin to update the system firmware.
+Until now only the version detection has been implemented.
+
+System identification
+---------------------
+
+coreboot can be detected the following ways:
+1. by parsing SMBIOS type 0 vendor string. On coreboot enabled platforms
+   it's always `coreboot`.
+2. by parsing ACPI tables. An ACPI device with the _HID `BOOT0000` exists.
+   (This HID has been reserved for coreboot enabled platforms)
+3. by parsing the devicetree. A node under *firmware/coreboot* with the
+   compatible id `coreboot` exists.
+
+coreboot version string
+-----------------------
+
+The coreboot version string always starts with `CBET`.
+After the prefix the *version*, *major*, *minor* string follows and finally
+the *build string*, containing the exact commit and repository state, follows.
+
+For example:
+> CBET4000 4.10-989-gc8a4e4b9c5-dirty
+
+GUID Generation
+---------------
+
+These device uses hardware ID values which are derived from SMBIOS.
+The following HWIDs are added on coreboot enabled platforms:
+
+* FU_HWIDS_KEY_MANUFACTURER +
+  FU_HWIDS_KEY_FAMILY +
+  FU_HWIDS_KEY_PRODUCT_NAME +
+  FU_HWIDS_KEY_PRODUCT_SKU +
+  FU_HWIDS_KEY_BIOS_VENDOR +
+  FU_HWIDS_KEY_BIOS_VERSION
+
+* FU_HWIDS_KEY_MANUFACTURER +
+  FU_HWIDS_KEY_FAMILY +
+  FU_HWIDS_KEY_PRODUCT_NAME +
+  FU_HWIDS_KEY_BIOS_VENDOR +
+  FU_HWIDS_KEY_BIOS_VERSION
+
+* FU_HWIDS_KEY_MANUFACTURER +
+  FU_HWIDS_KEY_FAMILY +
+  FU_HWIDS_KEY_PRODUCT_NAME +
+  FU_HWIDS_KEY_PRODUCT_SKU +
+  FU_HWIDS_KEY_BIOS_VENDOR
+
+* FU_HWIDS_KEY_MANUFACTURER +
+  FU_HWIDS_KEY_FAMILY +
+  FU_HWIDS_KEY_PRODUCT_NAME +
+  FU_HWIDS_KEY_BIOS_VENDOR
+
+They doesn't match the values provided by `fwupdtool hwids` or
```

Comment:
They doesn't?

Reply:
No.

Reply:
I think grammatically it should be "They don't"